### PR TITLE
Finish documenting whole testui package

### PR DESCRIPTION
**Pull request by paulsgrudups**
This PR contains changes to add docstrings to the whole testui package to provide documentation for the end-user and improve the linter score.

Small improvements were also made to improve the linter score like:
1) Code styling
2) Line length limit wrapping
3) `__contains__` conditions rewritten to `{value} in` format
4) Early function returns in places where within `if else` statements both scenarios return from function

### CODE REVIEW

**Comment by alvarolaserna** [comment]
> 
Can we add pylint job into the github actions?

File: testui/elements/testui_element.py [hunk 1]
```diff
@@ -118,68 +150,76 @@ def __find_by_element(self) -> WebElement:
...
-        if self.locator_type == "css":
-            return self.driver.find_elements(
-                by=By.CSS_SELECTOR, value=self.locator
-            )
-
-        if self.locator_type == "className":
-            return self.driver.find_elements(
-                by=By.CLASS_NAME, value=self.locator
-            )
-
```
Comment (TDL-EdgarsEglitis) [inline testui/elements/testui_element.py:154]: Is it ok to remove these methods?

**Comment by TDL-EdgarsEglitis** [comment]
It's probably worth checking whether the new return calls in `testui_driver.py` don't break anything.

**Comment by TDL-EdgarsEglitis** [comment]
Would this fully cover the implementation of #32?

**Comment by alvarolaserna** [comment]
> 

I think so, but we need to add a pylint job to the pipeline to keep things well formatted

File: testui/elements/testui_element.py (hunk 1 above)
Comment (alvarolaserna) [reply testui/elements/testui_element.py]: Was not okay, so I put them back

**Review by alvarolaserna — APPROVED** [review]

**Review by TDL-EdgarsEglitis — APPROVED** [review]
I suppose the remaining Pylint warnings should also be fixed eventually, but for now this looks good.

**Pull request merged**